editor.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const defaultEditor = "nano"
+
+// Cmd returns a *exec.Cmd editing the given path with $EDITOR or nano if no
+// $EDITOR is set.
+func editorCmd(path string) *exec.Cmd {
+	editor, args := getEditor()
+	return exec.Command(editor, append(args, path)...)
+}
+
+func getEditor() (string, []string) {
+	editor := strings.Fields(os.Getenv("EDITOR"))
+	if len(editor) > 1 {
+		return editor[0], editor[1:]
+	}
+	if len(editor) == 1 {
```

Comment:
if `EDITOR=""` then len(editor) will equal 1, so you need to check if `editor[0] == ""`

Reply:
Oh interesting.

Reply:
I tested with https://go.dev/play/p/3fSasMjhh-D

```
func main() {
	editor := strings.Fields("")
	if len(editor) > 1 {
		fmt.Println(editor[0], editor[1:])
	}
	if len(editor) == 1 {
		fmt.Println(editor[0], editor[1:])
	}
	fmt.Println("nano")
}
```

This code snippet outputs "nano" which is intended. I believe `strings.Fields` does return 0 when there is an empty string.